workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerCreation/SendCreatePlayerRequestSystem.cs

```diff
@@ -12,21 +13,50 @@ public class SendCreatePlayerRequestSystem : ComponentSystem
     {
         private readonly EntityId playerCreatorEntityId = new EntityId(1);
 
-        private struct Data
+        private struct SendData
         {
             public readonly int Length;
             [ReadOnly] public ComponentDataArray<PlayerCreator.CommandSenders.CreatePlayer> RequestSenders;
             [ReadOnly] public ComponentDataArray<WorkerEntityTag> DenotesWorkerEntity;
             [ReadOnly] public ComponentDataArray<OnConnected> DenotesJustConnected;
         }
 
-        [Inject] private Data data;
+        [Inject] private SendData sendData;
+
+        private struct ResponseData
+        {
+            public readonly int Length;
+            [ReadOnly] public ComponentDataArray<PlayerCreator.CommandResponses.CreatePlayer> Responses;
+            [ReadOnly] public ComponentDataArray<WorkerEntityTag> DenotesWorkerEntity;
+        }
+
+        [Inject] private ResponseData responseData;
 
         protected override void OnUpdate()
         {
-            var request = new CreatePlayerRequestType(new Vector3f(0, 0, 0));
-            data.RequestSenders[0].RequestsToSend
-                .Add(PlayerCreator.CreatePlayer.CreateRequest(playerCreatorEntityId, request));
+            for (var i = 0; i < sendData.Length; ++i)
+            {
+                var request = new CreatePlayerRequestType
+                {
+                    Position = new Generated.Improbable.Vector3f { X = 0, Y = 0, Z = 0 }
+                };
+
+                sendData.RequestSenders[i].RequestsToSend
+                    .Add(PlayerCreator.CreatePlayer.CreateRequest(playerCreatorEntityId, request));
+            }
+
+            for (var i = 0; i < responseData.Length; ++i)
+            {
+                foreach (var receivedResponse in responseData.Responses[i].Responses)
+                {
+                    Debug.Log($"Create player response code: {receivedResponse.StatusCode}");
```

Comment:
I don't think this log is necessary. If everything was successful - you will have a player, otherwise you get the error log below.

Reply:
aye